src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -172,16 +172,18 @@ internal string GetDebugString()
             return sb.ToString();
         }
 
-        /// <summary>
-        /// A helper method to help retrieve the connection string name for the configured storage provider.
-        /// </summary>
-        /// <returns>The connection string name for the configured storage provider.</returns>
-        public string GetConnectionStringName()
+        private void AppendStorageProviderValuesToDebugString(StringBuilder sb)
```

Comment:
Currently, this only logs what the customer's actually put in the host.json. It may be a better idea to make the factory responsible for logging, as then we get the default values of `AzureStorageOptions` logged as well.